FreeRTOS/Test/CMock/smp/Makefile

```diff
@@ -8,6 +8,7 @@ include ../makefile.in
 SUITES	+=	single_priority_no_timeslice
 SUITES	+=	single_priority_timeslice
 SUITES	+=	multiple_priorities_no_timeslice
+SUITES	+=	multiple_priorities_no_timeslice_1
```

Comment:
can you give it a meaningful name instead of _1, how is it different from the previous one